src/util/mod.rs

```diff
@@ -1,41 +1,15 @@
-use std::{io, slice};
+#[repr(C)] // guarantee 'bytes' comes after '_align'
+pub struct WorldAligned<Bytes: ?Sized> {
+    pub _align: [u32; 0],
```

Comment:
Is there any reason this struct can't have an explicit alignment attribute?

Reply:
good idea! I'll try that